opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/context/Xs2aContext.java

```diff
@@ -40,7 +40,7 @@
     private Map<ProtocolAction, String> flowByAction = ImmutableMap.of(
             ProtocolAction.LIST_ACCOUNTS, "xs2a-list-accounts",
             ProtocolAction.LIST_TRANSACTIONS, "xs2a-list-transactions",
-            ProtocolAction.INITIATE_PAYMENT, "xs2a-single-payments"
+            ProtocolAction.SINGLE_PAYMENT, "xs2a-single-payments"
```

Comment:
Just a note. Here it is OK to use `SINGLE_PAYMENT`

Reply:
ok